paddle/pten/kernels/impl/matmul_grad_kernel_impl.h

```diff
@@ -123,7 +123,7 @@ void MatMul(const Context& dev_ctx,
               b.data<T>(),
               mat_dim_b,
               static_cast<T>(1),
-              out->mutable_data<T>(),
+              out->data<T>(),
```

Comment:
replace with "mutable_data<T>(dev_ctx.GetPlace())"

Reply:
+1, Thanks for your review!